tests/benchmark/Int8Conv2dParallelBench.cpp

```diff
@@ -232,6 +232,9 @@ int main(int argc, char *argv[]) {
       "Usage: Int8Conv2dParallelBench numLayers(Int) "
       "numReps(Int) "
       "numAsyncLaunches(Int) numCores(Int) backendStr(String) dev_id(Int)\n");
+  printf("Standard Glow command-line options may be passed via the GLOW_OPTS "
```

Comment:
Quick question: These 3 lines are used in all those changes in this PR. Should there be a helper function to do it? 

Such a helper could be extended in the future to do more command-line options related logic in the future, should it be needed.

Reply:
Where should such a helper function live?

Also, I see some Facebook Internal checks have failed below but can't see what is failing to address it.

Reply:
Thinking some more about this: does this really make sense?  The two function calls that would be put in a helper function are:
- print help/usage information
- do LLVM argument setting via environment variable

Do we really want to bind these two things together?  It seems like it would be more fitting with standard practice to not always print the help/usage information, instead only printing it when requested.  The argument setting should be done always.

Reply:
ping?

Reply:
@geoffberry Sorry for the delay, missed this somehow. Are you suggesting to add a new flag to print usage explicitly, instead of always printing it? Moving the things you've added here to a helper function wouldn't change the functionality you've added, so just making sure we're talking about the same thing.

Reply:
@jfix71 Not exactly.  My point was that in my view there really should be two helper functions since the code I'm adding is doing two separate things:

- adding LLVM option processing
- printing a usage message describing how to use the above.

Always doing these two things together in the same helper function doesn't make sense to me.  Given that each of these things is a single function call in this change, I'm not sure how useful adding helper functions for them would be.

Reply:
Ok, I suppose I see them as always going together. Could just include a flag for whether to print the help info or not in the helper function. But I don't really mind either way, so since you feel strongly we shouldn't I'll just land as is.